docs/api/process.md

```diff
@@ -157,6 +158,21 @@ Returns `Object`:
 
 Returns an object with V8 heap statistics. Note that all statistics are reported in Kilobytes.
 
+### `process.getProcessMemoryInfo()`
+
+Returns `Object`:
+
+* `residentSet` Integer _Linux_ and _Windows_ - The amount of memory 
+currently pinned to actual physical RAM.
```

Comment:
I expect we'll get questions about why this metric isn't available on mac. Do you have any references you could add here about what a comparable statistic might be on macOS, or why this number isn't available on mac?